pypeman/persistence.py

```diff
@@ -79,6 +79,16 @@ async def get(self, namespace, key, default=SENTINEL):
         else:
             return self._data[namespace][key]
 
+    async def search_ids_by_value(self, namespace, value):
+        found_ids = []
+        for id, val in self._data[namespace].items():
+            if val == value:
+                found_ids.append(id)
+        return found_ids
+
+    async def table_lenght(self, namespace):
```

Comment:
get_table_length is a name that is very specific to database back ends.

couldn't you find a more generic name that might also be used for other back ends.

like

`get_num_entries()`


or just len()
you could even override the `self.__len__()` operator